pkg/restapi/vc/operation/operations.go

```diff
@@ -263,21 +261,35 @@ func (o *Operation) retrieveVCHandler(rw http.ResponseWriter, req *http.Request)
 		return
 	}
 
-	credentialResponse, err := o.client.ReadDocument(profile, id)
+	documentBytes, err := o.client.ReadDocument(profile, id)
 	if err != nil {
 		o.writeErrorResponse(rw, http.StatusBadRequest, err.Error())
 
 		return
 	}
 
-	_, err = rw.Write(credentialResponse)
+	document := operation.StructuredDocument{}
+
+	err = json.Unmarshal(documentBytes, &document)
+	if err != nil {
+		o.writeErrorResponse(rw, http.StatusInternalServerError,
+			fmt.Sprintf("structured document unmarshalling failed: %s", err.Error()))
+	}
+
+	responseMsg, err := json.Marshal(document.Content["message"])
+	if err != nil {
+		o.writeErrorResponse(rw, http.StatusInternalServerError,
+			fmt.Sprintf("structured document content marshalling failed: %s", err.Error()))
+	}
+
+	_, err = rw.Write(responseMsg)
```

Comment:
This change was made because I noticed that vc-service was returning a StructuredDocument, which is an EDV abstraction. I think vc-service should be returning the VC directly since the caller doesn't need to know about EDVs.